src/runtime/auto-prompt-manager.js

```diff
@@ -545,20 +526,40 @@ export class AutoPromptManager {
         return TYPE_REWARDED_SURVEY;
       }
 
-      if (potentialActions[0].type === TYPE_CONTRIBUTION) {
+      const contributionIndex = potentialActions.findIndex(
+        (action) => action.type === TYPE_CONTRIBUTION
+      );
+
+      if (contributionIndex > 0) {
+        actionToUse = potentialActions[0].type;
+        this.promptDisplayed_ = actionToUse;
+        return actionToUse;
+      }
+
+      // If the first potential action is contribution, and it has never been
```

Comment:
What if contribution isn't a potential action? (Ex: contributionIndex was -1)

Reply:
When contributionIndex is -1, it will be treated the same as when contributionIndex === 0.

Reply:
In that case, should the comment on line 539 change?

Reply:
Thanks!